### PR TITLE
Mjanowiecki patch 1

### PR DESCRIPTION
**Pull request by mjanowiecki**
## Purpose / why
  
Rearranges "Using Islandora" to help users find what they are looking for.

https://github.com/Islandora/documentation/issues/2154 

## What changes were made?

- Updated mkdocs. yml file
- Renamed md files to match new organization
- Moved md files to user-documentation if in "Using Islandora" section
- Fixed header hierarchies if necessary
- Changed titles to sentence case
- Fixed typos
- Fixed broken links

## Interested Parties

@Islandora/documentation

### Pull-request Reviewer
Pull-request reviewer should ensure the following:

* [x] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [x] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [x] Are the changes accurate, useful, free of typos, etc?

### Person Merging
The person merging should ensure the following:

* [x] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [x] If pages are renamed or removed, have all internal links to those pages been fixed?
* [x] If pages are added, have they been linked to or placed in the menu?
* [x] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Review by rosiel — APPROVED** [review]

File: docs/user-documentation/paged-content.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ resource node and _Page_ for the Islandora Model of the child resource node. Aft
 child resource nodes have "service file" media (either generated via the built-in derivative
 creation or added manually), they will be included in the OpenSeadragon paginated
 viewer displayed on the parent resource node's page.
-(See the [IIIF Documentation](../user-documentation/iiif.md#using-iiif-in-islandora-8)
+(See the [IIIF Documentation](../user-documentation/iiif.md/## Using IIIF in Islandora)
```
Comment (whikloj) [inline docs/user-documentation/paged-content.md:28]: Broken link, also the links seem to need the hyphens in the code. So I would change this back to the original.

File: docs/user-documentation/paged-content.md [hunk 2]
```diff
@@ -94,5 +94,4 @@ per file with that file attached as a Media. It is possible to select the node t
 This method does not create full metadata for the child nodes. It uses the filename as the node title.
  It also does not accept zip files; individual files must be uploaded. For more methods of bulk uploading content,
 see [Islandora Workbench](../technical-documentation/migration-islandora-workbench.md)
- and [Migrate Islandora CSV](../extending/#batch-upload-with-a-csv)
-
+ and [Migrate Islandora CSV](extending.md#Batch uploading via CSV)
```
Comment (whikloj) [inline docs/user-documentation/paged-content.md:97]: This brings you to the correct page, but not to the correct location. I think you need to leave the hyphens in for it to work. I would undo this change.

File: docs/user-documentation/context.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
 
 [Context](https://www.drupal.org/project/context) is a Drupal contrib module that allows configuration of "if this, then that" logic using an administrative user interface. Context defines "conditions" and "reactions" to enact the "if this [condition], then that [reaction]" logic. Some Islandora Legacy repositories use the community-contributed [Islandora Context](https://github.com/SFULibrary/islandora_context) module to apply this sort of logic to Islandora objects. An example Context from Islandora Legacy is
 
-> If an object's namespace is 'customnamespace', render the block containing the rights statement "Please contact Special Collections before reusing this item".
+> If an object's namespace is 'customnamespace', render the block containing the right's statement "Please contact Special Collections before reusing this item".
```
Comment (whikloj) [inline docs/user-documentation/context.md:5]: I have no English major and most of your changes were correct (even though I had to look up some of them). But this one seems odd. It is a "statement" of your "rights", not the "statement" possessed by the "right". But I could totally be wrong too.

File: docs/tutorials/create-a-resource-node.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ You are presented with a form showing a large number of metadata fields. Fill ou
 <!-- We should really include creating/adding a name here or in another tutorial!! It's a source of immediate confusion and frustration! -->
 
 
-Near the bottom of the form, the _System_ section contains mandatory fields that configure Islandora behaviours. Select **Image** from the _Model_ drop down box. This will enable image display behaviours, and cause image derivatives to be generated once we upload a file.
+Near the bottom of the form, the _System_ section contains mandatory fields that configure Islandora behaviours. Select **Image** from the _Model_ drop box. This will enable image display behaviours, and cause image derivatives to be generated once we upload a file.
```
Comment (whikloj) [inline docs/tutorials/create-a-resource-node.md:35]: I'm also not sure about this one, "drop down box" is the term for the web form element. This is definitely lingo, but I'm not sure that "drop box" is more clear?

File: docs/user-documentation/user-intro.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ Even though Fedora is optional in Islandora, most repositories will use it since
 * flexible, and configurable, disk storage architecture
 * fixity digest generation
 * Memento versioning
-* integration with RDF/Linked Data triplestores
+* integration with RDF/Linked Data triple-stores
```
Comment (whikloj) [inline docs/user-documentation/user-intro.md:38]: Should be triplestore as it refers to https://en.wikipedia.org/wiki/Triplestore. Maybe a link here would be helpful.

**Review by whikloj — APPROVED** [review]
The comments are for discussion and some are for my reference. Lots of great work here. Thank you @mjanowiecki
